app/Plugins/PluginManager.php

```diff
@@ -167,7 +167,7 @@ public function pluginExists(string $pluginName): bool
      */
     public function pluginEnabled(string $pluginName): bool
     {
-        return $this->getPlugin($pluginName)->plugin_active;
+        return (bool) optional($this->getPlugin($pluginName))->plugin_active;
```

Comment:
Looksie who loves optional :D

Reply:
It does a thing, just not the best thing.